### PR TITLE
cloudflare_exporter: adding initial code

### PR DESCRIPTION
**Pull request by wdauchy**
provides an initial cloudflare exporter

inspired from https://github.com/wehkamp/docker-prometheus-cloudflare-exporter
python3 compat
aiohttp handler

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+aiohttp
+cloudflare
```
Comment (cpaillet) [inline requirements.txt:2]: Need PR 76 Support API token , perhaps you can you add >= 2.3.1

File: cloudflare_exporter/exporter.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+from cloudflare_exporter.config import DEFAULT_HOST, DEFAULT_PORT
+from cloudflare_exporter.handlers import (handle_metrics,
+                                          handle_health)
+from cloudflare_exporter.collector import CloudflareCollector
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Cloudfalre prometheus exporter')
+    parser.add_argument('-t', '--token', type=str, required=True,
```
Comment (cpaillet) [inline cloudflare_exporter/exporter.py:14]: Seems the "token" only auth is new. You can parhaps add email+token one

File: requirements.txt (hunk 1 above)
Comment (wdauchy) [reply requirements.txt:2]: why? it was working properly on my local test

File: cloudflare_exporter/exporter.py (hunk 2 above)
Comment (wdauchy) [reply cloudflare_exporter/exporter.py:14]: ok that's something we can add in another review

File: requirements.txt (hunk 1 above)
Comment (cpaillet) [reply requirements.txt:2]: Yes, It's works now, because cloudflare just release 2.3.1. a few days ago.
but you can just add
clouflare >= 2.3.1

Comment (wdauchy) [reply requirements.txt:2]: see https://github.com/criteo/cloudflare-exporter/pull/3

**Pull request merged**